### ISSUE TITLE
missing key data o forwarding shared mailbox

### ISSUE THREAD

**Issue by larryrst** [MicrosoftDocs/microsoft-365-docs#3064]
The most common act of terminating an employee is the need to have future mail go to another employee.  This article is good, but does not detail how best to do that.  I believe that that process should be after changing the password or blocking the user: convert the user to a shared mailbox; 2) turn forwarding on for the shared mailbox, and chose to not leave a copy (optional); 3) remove license (depending on how conversion was done).

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 4e366b9c-0d35-83c6-f509-1774dd351504
* Version Independent ID: 702f2dc1-46f8-508a-68bc-b5d033c56a9a
* Content: [Remove a former employee - Microsoft 365 admin](https://docs.microsoft.com/en-us/microsoft-365/admin/add-users/remove-former-employee?view=o365-worldwide)
* Content Source: [microsoft-365/admin/add-users/remove-former-employee.md](https://github.com/MicrosoftDocs/microsoft-365-docs/blob/public/microsoft-365/admin/add-users/remove-former-employee.md)
* Service: **o365-administration**
* GitHub Login: @KwekuA
* Microsoft Alias: **kwekua**

**Comment by msbemba** [comment]
@larryrst Thank you for your feedback. I hear what you are saying, but is that not covered under the section - Forward a former employee's email to another employee or convert to a shared mailbox? - https://docs.microsoft.com/en-us/microsoft-365/admin/add-users/remove-former-employee?view=o365-worldwide#forward-a-former-employees-email-to-another-employee-or-convert-to-a-shared-mailbox

Let me know if you think we need to add more information.
Thanks
Sri

**Comment by larryrst** [comment]
Hi Sri,

No, I don’t think that it is, or more accurately, it is wrong in its statement implications.  That article states that “Email forwarding requires that the former employee's account has a license.”  This is wrong.  I can (and should) forward the email of a shared mailbox to another user after terminating a valuable employee, after converting them to a shared mailbox.  This is done without a license.

The article does say that I can forward email TO a shared mailbox, but not from one.

In summary, I feel that the instructions on how to implement one of the primary ways to terminate and use shared mailboxes  is missing.

Note:  if the mailbox is larger than 50GB, the license need statement is true.

-Larry

**Comment by yogkumgit** [comment]
@msbemba Thank you very much for the contribution and sharing this explanation. @larryrst Hope this comment is helpful for you. If you see a documentation update is required, please feel free to open an issue for the same. We proceed here to close it. Thanks for taking out some time to open the issue. Appreciate and encourage you to do the same in future also.

**Comment by msbemba** [comment]
@larryrst That is correct. You don't need a license to set up forwarding on a shared mailbox. I have made both the changes in the document via PR#https://github.com/MicrosoftDocs/microsoft-365-docs/pull/3120

Thanks
Sri

**Comment by yogkumgit** [comment]
@msbemba Thank you very much for the contribution and updating the documentation with PR. @larryrst Hope this update is helpful for you. Thanks for taking out some time to open the issue. Appreciate and encourage you to do the same in future also.
